src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -57,21 +58,27 @@ function ExportAnnotations({
   };
 
   return (
-    <>
+    <form
+      className="space-y-3"
+      onSubmit={exportAnnotations}
+      data-testid="export-form"
+    >
       {exportCount > 0 ? (
         <>
-          <p data-testid="export-count">
+          <label data-testid="export-count" htmlFor="export-filename">
             Export{' '}
             <strong>
               {exportCount} {pluralize('annotation', exportCount)}
             </strong>{' '}
             in a file named:
-          </p>
+          </label>
           <Input
             data-testid="export-filename"
             id="export-filename"
             defaultValue={suggestedFilename({ groupName: group?.name })}
             elementRef={inputRef}
+            required
+            maxLength={255}
```

Comment:
Since the ".json" suffix will take up 5 chars, we should reduce this to be <= 250 at least.

When testing in Safari on macOS, I noticed that downloads would start and then fail if I tried to use all 255 chars. In Chrome the problem did not occur as Chrome truncates the entered filename as needed. In Safari removing the last few characters allowed the download to succeed again.

Reply:
Good catch!